tools/docs/config.ts

```diff
@@ -271,11 +271,18 @@ export async function generateConfig(dist: string, bot = false): Promise<void> {
 }
 
 function generateAdvancedUse(): string {
-  return '\n<!-- prettier-ignore -->\n!!! warning\n    For advanced use only! Use at your own risk!\n';
+  return (
+    '\n<!-- prettier-ignore -->\n!!! warning\n' +
+    '    For advanced use only! Use at your own risk!\n'
+  );
 }
 
 function generateFollowTagAdvancedFunctionality(): string {
-  return "\n<!-- prettier-ignore -->\n!!! warning\n    Advanced functionality.\n    Only use this if you're sure you know what you're doing.\n";
+  return (
+    '\n<!-- prettier-ignore -->\n!!! warning\n' +
+    '    Advanced functionality.\n' +
+    "    Only use this if you're sure you know what you're doing.\n"
```

Comment:
```suggestion
    '    Only use this if you're sure you know what you're doing.\n'
```

Do you really mean to use `"` here? The other lines use `'`.